coalib/parsing/DefaultArgParser.py

```diff
@@ -249,6 +249,15 @@ def default_arg_parser(formatter_class=None):
         ' additional arguments. If used without arguments, all bears specified'
         ' with --bears will be debugged (even implicit dependency bears).')
 
+    outputs_group.add_argument(
+        '--profile', nargs='?', const=True,
+        help='Enable bear profiling with cProfile, that can help to create'
+        ' performant bears by profiling the Bear code. To specify where to'
+        ' dump the profile files, supply the path or the directory name, all'
+        ' bears (even implicit dependency bears) in a section will be profiled.'
+        ' Profile data will be save in the specified directory or the path'
+        ' with name format {section.name}_{bear.name}.prof.')
```

Comment:
Needs a bit rephrasing, since what is specified here has to be always a directory (a path in general can also be a file). Especially worth to mention is that if it doesn't exist, the directory will be created.